Removing GRAIL references

rj3d opened the pull request:
@siddharthab could you look this over?

Engineering leadership and legal here at GRAIL have given us the okay to transfer ownership of the repository, but they want all references to GRAIL scrubbed first. Could you look this over to me to make sure everything looks good? Tests are currently broken for unrelated reasons, and I don't have the bandwidth to fix them at the moment. Can you run any tests that you'd like to make sure I didn't break anything?

Commit:
Removing grail references.

Commit:
Remove grail references from image test.

Commit:
Remove GRAIL R style guide.

Commit:
More renaming.

siddharthab commented:
Thank you! I can take care of the remaining work for this. Is there going to be some similar effort for the other bazel-* repos in the grailbio org?

rj3d commented:
@siddharthab do you know if I can safely rename the importpath in [gazelle/BUILD](https://github.com/grailbio/rules_r/blob/grail-rename/gazelle/BUILD)? If so, do you have any suggestions as to what it could be renamed? Thanks!

siddharthab commented:
> @siddharthab do you know if I can safely rename the importpath in [gazelle/BUILD](https://github.com/grailbio/rules_r/blob/grail-rename/gazelle/BUILD)? If so, do you have any suggestions as to what it could be renamed? Thanks!

Yes, you can rename it to `bazel.io/gazelle/rlang`. It will not be a breaking change.

rj3d commented:
@siddharthab awesome, thank you! 

I hadn't heard any discussion of transferring the other bazel-* repos in the grailbio org. Would you mind listing out all of the ones that you think would be useful to transfer, and I can ask to confirm that it would be okay?

siddharthab commented on R/container/README.md:
````diff
@@ -33,22 +32,23 @@ container_pull(
 
 ```python
-load("@com_grail_rules_r//R:defs.bzl", "r_library")
+load("@com_rules_r//R:defs.bzl", "r_library")
````
Can we rename these to `rules_r` instead?

rj3d replied:
That's fine with me!

siddharthab commented on LANGUAGE.md:
Does this file need to be deleted? We can remove the top paragraph. As far as I know, this is the last remaining source of this file.

rj3d replied:
No reason it all has to be deleted. I added it back and removed the top paragraph.

Commit:
Rename gazelle importpath.

rj3d commented:
> Yes, you can rename it to `bazel.io/gazelle/rlang`. It will not be a breaking change.

Done! Thank you.

siddharthab commented on tests/container/BUILD:
```diff
@@ -22,17 +22,17 @@ r_library_tar(
     ],
     library = "//:library",
-    library_path = "grail/r-libs",
+    library_path = "R/r-libs",
```
Can you rename this to `rules_r` instead of just R?

siddharthab commented on tests/container/BUILD:
```diff
@@ -22,17 +22,17 @@ r_library_tar(
     name = "library_image",
     base = "@r_base//image",
-    directory = "grail",
+    directory = "r",
```
Same here. `rules_r` please instead of `r`.

rj3d replied:
Done!

siddharthab commented on README.md:
```diff
@@ -41,10 +45,11 @@ Bazel, over a custom solution of tracking the package dependency graph and
 and tested as part of one build system in multi-language monorepos.
 
-These rules are mature for production use. We use these rules internally at
+These rules are mature for production use. These rules are used at
```
If you want, we can remove the second sentence here completely.

siddharthab commented:
> Would you mind listing out all of the ones.

Sure. Thank you. The others are:
https://github.com/grailbio/bazel-toolchain
https://github.com/grailbio/bazel-compilation-database

Once approved, I can repeat the work you did for this repo and send you a review, so that it's easier for you.

siddharthab commented:
If you want, I can take over this PR and finish addressing all the comments I left and fix the tests.

rj3d commented:
> If you want, I can take over this PR and finish addressing all the comments I left and fix the tests.

That would be great! Is there anything you need from me too accomplish that?

Commit:
Code review changes.

Commit:
Rename docker R folder to rules_r

Commit:
Remove GRAIL usage reference from readme.

rj3d commented:
> 
> Sure. Thank you. The others are: https://github.com/grailbio/bazel-toolchain https://github.com/grailbio/bazel-compilation-database
> 
> Once approved, I can repeat the work you did for this repo and send you a review, so that it's easier for you.

I just asked to see what management thinks. I'll let you know when I hear back.

siddharthab commented:
The test failures are unrelated. I am going to merge your changes first and then work on fixing the test failures. Thank you so much for all your help!
